BUILDIT.md

```diff
@@ -295,7 +294,7 @@ end
 Now if you refresh the page 
 you should see the following:
 
-![liveveiw-page-with-tailwind-style](https://user-images.githubusercontent.com/194400/176137805-34467c88-add2-487f-9593-931d0314df62.png)
```

Comment:
Markdown syntax for images is fine as the M⬇️  parser will render it to HTML. 👌 

Also, consider constraining the viewport to the minimum size to reduce bandwidth and scrolling.
e.g:
![liveview](https://user-images.githubusercontent.com/17494745/198072436-71ab8739-e4c7-4cdb-84f0-3803cc2f1fcc.png)

Could be reduced by 60% to:
<img width="1077" alt="liveview" src="https://user-images.githubusercontent.com/194400/198254445-df7053be-dcc6-4899-875e-d294fc1ac5d8.png">

Sill gives people the exact same info but doesn't consume their bandwidth/time.

Reply:
E.g: 
![liveview-cropped](https://user-images.githubusercontent.com/194400/198260922-c49b9561-5900-4b58-a57c-5e4d984f5e5a.png)